CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ### Next
 
+### 1.4.4
```

Comment:
This is a new feature, it would be 1.5.0 🚀

Reply:
By default, I thought this repo follow react-native semantic versioning and for breaking changes increase minor version. If this repo follows the global semantic versioning I'll change the version to 1.5.0. 
Done 👍

Reply:
No - we did that before we went 1.0, but I personally don't like that react-native is used in production but has the semantics of their releases crammed into 2 places by not using the major version. I much prefer semantic versioning so we're using it now. Thank you!